console/src/components/Main/Main.js

```diff
@@ -86,17 +86,25 @@ class Main extends React.Component {
   setShowUpdateNotification() {
     const {
       latestStableServerVersion,
-      latestServerVersion,
+      latestPreReleaseServerVersion,
       serverVersion,
       console_opts,
     } = this.props;
 
     const allowPreReleaseNotifications =
       !console_opts || !console_opts.disablePreReleaseUpdateNotifications;
 
-    const latestServerVersionToCheck = allowPreReleaseNotifications
-      ? latestServerVersion
-      : latestStableServerVersion;
+    let latestServerVersionToCheck;
+    if (allowPreReleaseNotifications) {
+      latestServerVersionToCheck = versionGT(
+        latestPreReleaseServerVersion,
+        latestStableServerVersion
+      )
+        ? latestPreReleaseServerVersion
+        : latestStableServerVersion;
+    } else {
+      latestServerVersionToCheck = latestStableServerVersion;
+    }
```

Comment:
One small suggestion. This ternary looks less readable since it's multiline. 

```js
    let latestServerVersionToCheck;
    const isVersionGT = versionGT(latestPreReleaseServerVersion, latestStableServerVersion);
    if (allowPreReleaseNotifications && isVersionGT) {
      latestServerVersionToCheck = latestPreReleaseServerVersion
    } else {
      latestServerVersionToCheck = latestStableServerVersion;
    }
```

Reply:
Yeah. This was auto-formatted. I genuinely hate this auto-formatiing of code that happens. Would really like to disable it. Makes a lot of readable code much less readable.

Will make this an if-else

Reply:
Sorry to be annoying but I still don't like that these parts are the same 😛
<img width="484" alt="Screenshot 2020-02-24 at 12 41 15" src="https://user-images.githubusercontent.com/9019397/75134028-02b29d80-5703-11ea-8af5-2d6e6ac20277.png">

Reply:
Done

Reply:
> Sorry to be annoying but I still don't like that these parts are the same

Fixed :)